apps/frontend/src/utils/aave/AaveUserReservesSummary.ts

```diff
@@ -228,13 +221,14 @@ export class AaveUserReservesSummaryFactory {
             suppliedUSD: Decimal.from(r.underlyingBalanceUSD),
             borrowed: Decimal.from(r.totalBorrows),
             borrowedUSD: Decimal.from(r.totalBorrowsUSD),
-            availableToBorrow,
-            availableToBorrowUSD,
+            availableToBorrow: availableLiquidity.lt(canBorrow)
+              ? availableLiquidity
+              : canBorrow,
 
             borrowRateMode: Decimal.from(r.variableBorrows).gt(0)
               ? BorrowRateMode.VARIABLE
               : BorrowRateMode.STABLE,
-          };
+          } as ReserveSummary;
```

Comment:
If the cast arised from removing the field availableToBorrowUSD, maybe the type ReserveSumary should be changed instead of casting